client/room.html

```diff
@@ -5,10 +5,18 @@
   <meta charset="utf-8">
   <title>VirtualTabletop.io</title>
   <meta name="viewport" content="width=device-width, initial-scale=1.0, maximum-scale=1.0, user-scalable=0">
+
+  <link rel="icon"
+  href="https://cdn.glitch.com/12918bef-605d-465b-83e9-daddde8072e9%2Fw.png?1553040982360"
+  id="light-scheme-icon">
+<link rel="icon"
+  href="https://cdn.glitch.com/12918bef-605d-465b-83e9-daddde8072e9%2Fb.png?1553040985667"
+  id="dark-scheme-icon">
```

Comment:
What's the point of these? I was wondering why I saw completely unrelated favicons on my tab.

Reply:
This was a mistake on my part as I copy+pasted some things to fiddle with how they did it and I accidentally left them in. I am removing them now.